Insolvency.CalculationsEngine.Redundancy.API/Infrastructure/Middlewares/Validators/APPACalculationRequestValidator.cs

```diff
@@ -34,12 +34,12 @@ public APPACalculationRequestValidator()
 
             RuleFor(req => req)
                .Must(RP1DataPresent)
-               .WithMessage($"No Arrears Of Pay RP1 data has been not provided")
+               .WithMessage($"Arrears Of Pay RP1 data has been not provided")
                .When(req => req.Ap != null);
 
             RuleFor(req => req)
                 .Must(RP14aDataPresent)
-                .WithMessage($"No Arrears Of Pay RP14a data has been not provided")
+                .WithMessage($"Arrears Of Pay RP14a data has been not provided")
```

Comment:
Is this wording correct?  "Arrears Of Pay RP14a data has been not provided"   Should it not be "Arrears Of Pay RP14a data has not been provided"??

Reply:
Good spot @IanCarson10 . Thanks for letting us know.